.github/workflows/ios.yml

```diff
@@ -0,0 +1,43 @@
+name: MacOS
+
+# https://www.jeffgeerling.com/blog/2020/running-github-actions-workflow-on-schedule-and-other-events
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * SUN'
+
+jobs:
+  build-ios:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-10.15, macos-11.0]
+
+    steps:
+    # https://github.com/marketplace/actions/cancel-workflow-action
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
```

Comment:
```suggestion
      uses: styfle/cancel-workflow-action@0.9.1
```

For latest version.

Reply:
Yes, I just saw your change in the other workflow files